src/applications/disability-benefits/all-claims/pages/toxicExposure/toxicExposureChoicePage.jsx

```diff
@@ -0,0 +1,358 @@
+import React, { useRef, useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import FormNavButtons from 'platform/forms-system/src/js/components/FormNavButtons';
+import {
+  VaCheckboxGroup,
+  VaModal,
+  VaAlert,
+} from '@department-of-veterans-affairs/component-library/dist/react-bindings';
+import { scrollAndFocus, scrollToFirstError } from 'platform/utilities/ui';
+import cloneDeep from 'platform/utilities/data/cloneDeep';
+import {
+  conditionsDescription,
+  conditionsPageTitle,
+  conditionsQuestion,
+  makeTEConditionsUISchema,
+} from '../../content/toxicExposure';
+import { formTitle } from '../../utils';
+import {
+  deletedToxicExposureAlertConfirmationContent,
+  deleteToxicExposureModalContent,
+  deleteToxicExposureModalDescription,
+  deleteToxicExposureModalTitle,
+} from '../../content/toxicExposureChoiceContent';
+
+export const toxicExposureChoicePageTitle =
+  'Option to claim conditions related to toxic exposure';
+
+/**
+ * List of all toxic exposure related keys that need to be deleted
+ * when user opts out of toxic exposure claims.
+ * @constant {string[]}
+ */
+export const toxicExposureKeys = [
+  'conditions',
+  'gulfWar1990',
+  'gulfWar1990Details',
+  'gulfWar2001',
+  'gulfWar2001Details',
+  'herbicide',
+  'herbicideDetails',
+  'herbicideOtherLocations',
+  'additionalExposures',
+  'additionalExposuresDetails',
+  'specifyOtherExposures',
+];
+
+export const sectionsOfToxicExposure = [
+  'Toxic exposure conditions',
+  'Gulf War service locations and dates (1990 and 2001)',
+  'Agent Orange exposure locations and dates',
+  'Other toxic exposure details and dates',
+];
+
```

Comment:
This list and the content of `deleteToxicExposureModalContent` in `src/applications/disability-benefits/all-claims/content/toxicExposureChoiceContent.jsx `appear to be identical. It may make sense for each to have a shared source, in case any future changes are made to content.